bindings/container.cpp

```diff
@@ -177,6 +194,6 @@ App* Container::loadAndInstantiate(QString path) {
         std::cout << QString("%1 is not a valid Holochain DNA file!").arg(path).toStdString() << std::endl;
         return 0;
     } else {
-        return new App(dna, this);
+        return new App(dna, QStandardPaths::writableLocation(QStandardPaths::TempLocation), this);
```

Comment:
should this be `QStandardPaths::AppDataLocation`? From the docs:

> Returns a directory location where persistent application data can be stored. This is an application-specific directory.

Reply:
I'm using `AppConfigLocation` above.
This function is for transient instances and for usage in tests. But I have to provide some path in that case as well. But it should be temporary space here.

Reply:
Ah, cool! marking as approved, then : )